File: test/static/lang/java/SwitchDefaultOpen.java
class SwitchDefaultOpen{
   public static void main(String args[]){
      String monthString;
      switch (month) {
         case 1:  monthString = "January";
                  break;
         case 12: monthString = "December";
                  break;
      }

      switch (month) {
         case 1:  monthString = "January";
                  break;
         case 12: monthString = "December";
                  break;
         /*
         default: monthString = "Invalid month";
                  break;
                  */
      }

      switch (month) {
         case 1:  monthString = "January";
                  break;
         case 12: monthString = "December";
                  break;
         //default: monthString = "Invalid month";
                  //break;
      }

   }
}

File: test/static/lang/csharp/SwitchDefaultOpen.cs
using System;

public class SwitchDefaultOpen{
   public static void Main()
   {

      byte[] bytes1 = new byte[100];
      byte[] bytes2 = new byte[100];
      var rnd1 = new Random();
      Random rnd2 = new Random();

      rnd1.NextBytes(bytes1);
      rnd2.NextBytes(bytes2);

      RijndaelManaged rjndl = new RijndaelManaged();
      rjndl.KeySize = 64;
      rjndl.BlockSize = 256;
      rjndl.Mode = CipherMode.ECB;
      ICryptoTransform transform = rjndl.CreateEncryptor();
      Console.WriteLine("Test Console")
      Debug.WriteLine("Test Debug")

      string monthString;
      switch (month)
      {
         case 1:
            monthString = "January";
            break;
         case 12:
            monthString = "December";
            break;
      }

      switch (month)
      {
         case 1:
            monthString = "January";
            break;
         case 12:
            monthString = "December";
            break;
         /*
         default:
            monthString = "Invalid month";
            break;
         */
      }

      switch (month)
      {
         case 1:
            monthString = "January";
            break;
         case 12:
            monthString = "December";
            break;
         //default:
            //monthString = "Invalid month";
                  //break;
      }
   }
}
